lib/PHPExif/Adapter/Native.php

```diff
@@ -345,4 +358,30 @@ public function mapData(array $source)
 
         return $mappedData;
     }
+
+    /**
+     * Extract GPS coordinates from components array
+     *
+     * @param array $components
+     * @return float
+     */
+    protected function extractGPSCoordinate(array $components)
+    {
+        $components = array_map(array($this, 'normalizeGPSComponent'), $components);
+
+        return intval($components[0]) + (intval($components[1]) / 60) + (floatval($components[2]) / 3600);
+    }
+
+    /**
+     * Normalize GPS coordinates components
+     *
+     * @param mixed $component
+     * @return int|float
+     */
+    protected function normalizeGPSComponent($component)
```

Comment:
Can you unit test this method? Coverage is brought down now, which I want to avoid.